felix/bpf/mock/multiversion/map.go

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2022 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multiversion
+
+import (
+	"github.com/projectcalico/calico/felix/bpf"
+	"github.com/projectcalico/calico/felix/bpf/bpfmap/upgrade"
+	v2 "github.com/projectcalico/calico/felix/bpf/mock/multiversion/v2"
+	v3 "github.com/projectcalico/calico/felix/bpf/mock/multiversion/v3"
+	v4 "github.com/projectcalico/calico/felix/bpf/mock/multiversion/v4"
+	v5 "github.com/projectcalico/calico/felix/bpf/mock/multiversion/v5"
+)
+
+func GetMapParams(version int) bpf.MapParameters {
```

Comment:
This is fine for the test, I think, for a real map, the package may contain some array of sort, maybe not, idk :man_shrugging: